reflex/app.py

```diff
@@ -493,16 +493,30 @@ def get_frontend_packages(self, imports: Dict[str, str]):
         Example:
             >>> get_frontend_packages({"react": "16.14.0", "react-dom": "16.14.0"})
         """
-        page_imports = [
+        page_imports = {
             i
             for i in imports
             if i not in compiler.DEFAULT_IMPORTS.keys()
             and i != "focus-visible/dist/focus-visible"
             and "next" not in i
             and not i.startswith("/")
             and i != ""
-        ]
-        page_imports.extend(get_config().frontend_packages)
+        }
+        frontend_packages = get_config().frontend_packages
```

Comment:
Should we move these warnings to the `Config.__init__` method?

Reply:
We don't have access to the inferred packages in Config.__init__, so I'd say no.